base_layer/wallet/src/base_node_service/mock_base_node_service.rs

```diff
@@ -100,6 +105,7 @@ impl MockBaseNodeService {
             is_synced: Some(true),
             updated: None,
             latency: None,
+            status: OnlineStatus::default(),
```

Comment:
Not sure that a `Default` impl makes sense for `OnlineStatus`, by reading this it's not clear what is it set to. I think rather be explicit:
```suggestion
            status: OnlineStatus::Connecting
```